### PR TITLE
Follow gene

### PR DESCRIPTION
**Pull request by xtina-starr**
This PR adds follow gene mutation so we can implement https://github.com/artsy/collector-experience/issues/506:

screen shot 2017-11-29 at 3 25 07 pm

### CODE REVIEW

File: lib/loaders/loaders_without_authentication/gravity.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ export default requestIDs => {
     fairsLoader: gravityLoader("fairs"),
     geneFamiliesLoader: gravityLoader("gene_families"),
     matchGeneLoader: gravityLoader("match/genes"),
+    geneLoader: gravityLoader(id => `/gene/${id}`),
```
Comment (damassi) [inline lib/loaders/loaders_without_authentication/gravity.js]: Maybe move this item up so that the list remains alphabetical?

Comment (xtina-starr) [reply lib/loaders/loaders_without_authentication/gravity.js]: good call!

File: schema/me/__tests__/follow_gene.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  it("follows a gene", () => {
+    const mutation = `
+      mutation {
+        followGene(input: { gene_id: "pop-art" }) {
+          gene {
+            id
+            name
+          }
+        }
+      }
```
Comment (damassi) [inline schema/me/__tests__/follow_gene.test.js]: Its funny how straight forward mutations are compared to what I used to have in my mind. Did they become simpler at some point? I remember there used to be an `input` type and a lot of strange ceremony, but yah! so nice.

**Comment by damassi** [comment]
LGTM 👍  will merge on 🍏

**Comment by xtina-starr** [comment]
We did have one question around whether we should support the unfollow similar to the [followArtist](https://github.com/artsy/metaphysics/blob/master/schema/me/follow_artist.js#L8) mutation. We are adding this for onboarding and there isn't a need to unfollow and we personally would have expected unfollow to perhaps be a separate mutation (but we are new around here in MP world).

File: schema/me/__tests__/follow_gene.test.js (hunk 2 above)
Comment (orta) [reply schema/me/__tests__/follow_gene.test.js]: The "input" and strange ceremony (the mutation ID etc) was a requirement of "classic relay"

**Pull request merged**